soliket/ccl.py

```diff
@@ -175,3 +200,61 @@ def get_CCL(self):
         :return: dict of results
         """
         return self._current_state['CCL']
+
+class Tester(Likelihood):
+    # Cross and auto data
+
+    auto_file: str = 'input/clgg_noiseless.txt'
```

Comment:
what are these paths relative to?  See some of the other likelihoods for how to package data.  Options might be making it installable, or including the data in the package and finding it with `resource_filename`.